README.md

```diff
@@ -549,7 +572,7 @@ your `PATH` or [define an environment variable](#using-a-different-pager). The [
 ### Colors
 
 Windows 10 natively supports colors in both `conhost.exe` (Command Prompt) and PowerShell since
-[v1511](https://en.wikipedia.org/wiki/Windows_10_version_history#Version_1511_(November_Update)), as
+[v1511](<https://en.wikipedia.org/wiki/Windows_10_version_history#Version_1511_(November_Update)>), as
```

Comment:
Although that worked fine before, that's pretty neat. I didn't know Jekyll/GitHub supported escaping URLs with parentheses like that.